python-threatexchange/threatexchange/tests/test_file_content_type.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from threatexchange.content_type.photo import PhotoContent
+from threatexchange.content_type.video import VideoContent
+from threatexchange.content_type.file_content import FileContent
+
+class TestFileContentType(unittest.TestCase):
```

Comment:
blocking: In this library, we use py.test and not unittest.TestCase. Take look at some of the other tests in this file to see how to write those.

The logic overall looks good though!

Reply:
I can't find any tests that using py.test 
All are using unittest.TestCase
@Dcallies